x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/AbstractSqlRequest.java

```diff
@@ -56,8 +58,9 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         out.writeEnum(requestInfo.mode());
         out.writeOptionalString(requestInfo.clientId());
+        out.writeOptionalString(requestInfo.version() == null ? null : requestInfo.version().toString());
```

Comment:
since you use `writeOptionalString` no need for null check.

Reply:
the check is there for the `.toString()` to be safe.

Reply:
ah yep, I see, thx!